.eslintrc.js

```diff
@@ -111,6 +111,7 @@ module.exports = {
     "prefer-const": "error",
     quotes: ["error", "double"],
     semi: "off",
+    "no-restricted-imports": ["error", {patterns: ["../lib/*", "@chainsafe/*/lib/*"]}],
```

Comment:
I'm pretty sure we have a lot of places where we need "@chainsafe/*/lib/*" because not everything is exported at index.

Reply:
You can see the list in this PR's lint job, it's not too long now. With the root imports in the types and config packages most of the necessary usages are covered. Then it's mostly tests utils and stubs. We can add an
```ts
/* eslint-disable no-restricted-imports */
```
line there. The goal of this is to prevent unnecessary / un-intended usage of lib imports. Happy to hear other proposals